### DISCUSSION TITLE
Stops at the stage of obtaining IP on ZTE A515

### DISCUSSION THREAD

**Discussion by fdanext** [Mygod/VPNHotspot#363] [Q&A]
Tell me, I can not connect? 
I tried different Android 4.0 - 11 devices to connect, different VPN services - the result is the same, it freezes at the stage of obtaining IP.
Maybe the interface is incorrect.
Internet is mobile.

![Screenshot_2022-04-13-02-33-23](https://user-images.githubusercontent.com/102393215/163072650-a4aef0a7-235a-46b4-aa06-2897775e79d4.png)
![Screenshot_2022-04-13-02-33-54 — копия](https://user-images.githubusercontent.com/102393215/163072651-7fbbaba7-736b-409d-a0ef-c3596d1326d3.png)

**Comment by Mygod** [comment]
Try system hotspot + toggling the interface in the app afterwards?

**Comment by ale1976-code** [comment]
> Tell me, I can not connect? I tried different Android 4.0 - 11 devices to connect, different VPN services - the result is the same, it freezes at the stage of obtaining IP. Maybe the interface is incorrect. Internet is mobile.
> 
> ![Screenshot_2022-04-13-02-33-23](https://user-images.githubusercontent.com/102393215/163072650-a4aef0a7-235a-46b4-aa06-2897775e79d4.png) ![Screenshot_2022-04-13-02-33-54 — `[](копия](https://user-images.githubusercontent.com/102393215/163072651-7fbbaba7-736b-409d-a0ef-c3596d1326d3.p[](url))`

**Comment by ale1976-code** [comment]
``
